src/main/java/org/openrefine/extensions/commons/utils/CommonsImportingController.java

```diff
@@ -190,74 +191,41 @@ private static void parsePreview(
             ObjectNode options,
             List<Exception> exceptions) throws IOException {
 
-        String cmtitle = "Category:" + JSONUtilities.getString(options, "docUrl", null);
-        String url = "https://commons.wikimedia.org./w/api.php"
-                + "?action=query&list=categorymembers&cmtitle="
-                + cmtitle + "&cmtype=subcat|file&cmprop=title|type|ids&cmlimit=5&format=json";
-        OkHttpClient client = new OkHttpClient.Builder().build();
-        Request request = new Request.Builder().url(url).build();
-        Response response = client.newCall(request).execute();
-        JsonNode jsonNode = new ObjectMapper().readTree(response.body().string());
-        JsonNode files = jsonNode.path("query").path("categorymembers");
         JSONUtilities.safePut(options, "headerLines", 0);
+        String cmtitle = JSONUtilities.getString(options, "docUrl", null);
```

Comment:
The `docUrl` parameter seems to be inherited from the Google Data extension but does not make much sense in this context, right? You could call it `category` instead, perhaps?

Reply:
changed it to `categoryInput` but didn't amend the commit, updating soon